src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/styles/common';
+
+.standardLabelValue {
+    display: grid;
+    grid-template-columns: [label] 120px 10px [value] auto;
```

Comment:
In this case, where we only have two columns, better define the space between them using grid-column-gap. Makes the intent clearer.

Reply:
Updated it to use `grid-column-gap`. 
Regarding the second point, I think we can look into better ways to structure it when we have more contents to add in.